command/ca/provisionerbeta/add.go

```diff
@@ -217,6 +217,8 @@ provisioning tokens.`,
 			awsAccountFlag,
 			azureTenantFlag,
 			azureResourceGroupFlag,
+			azureSubscriptionIDFlag,
+			azureObjectIDFlag,
```

Comment:
I don't see these flags being added to the provisioner details in the `createAzureDetails` function.

Reply:
Apologies I missed updating these - I have now updated the two functions and bumped the linkedca dependency.